### ISSUE TITLE
Request/Help: Simulate downloaded torrent; tell radarr which available torrent you downloaded

### ISSUE THREAD

**Issue by Zachary24** [Radarr/Radarr#3316]
**Is your feature request related to a problem? Please describe.**
I'll try to make this a quick one. Initially my naming scheme was messed up (had no clue till emby started messing up), radarr would name stuff like _Venom - (2018) - -_ after fixing that I changed the scheme to Venom (2018), after making radarr renaming all files and folders radarr wasn't noticing them as the set path for SOME (not all) was still the old scheme (_Venom - (2018) - -_) after hours of searching I figured out the only way to get the expected paths correct was to simply remove all of the movies I have, continue working on getting all of the movies I still need, once I get everything, nuke every movie, and then import them all from my disk, so that the paths would all be correct, (I've looked in both static and dynamic paths and changing them had no effect), after all of that I decided I'd go ahead and use a different scheme since I'll have to start from the beginning again _Venom (2018) - (Bluray-1080p Proper) - (RARBG)_, but now I'm realizing after deleting the movies from radarr it won't show/acknowledge which torrent it came from, however just by looking at (a) seed count and (b) filesize it's pretty easy for me to figure out which one it is. So is there any way to tell radarr which torrent should be associated with that film? Just like how Emby has the Identify option, and like plex has fix match.

**TL;DR:
Have to nuke all films, since I changed my naming scheme (and radarr won't change the expected path). So is there any way to tell radarr which torrent should be associated with that film? Instead of having to re-download the torrent. Just like how Emby has the Identify option, and like plex has fix match**

**Comment by fryfrog** [comment]
You don't need to associate old old torrents w/ your movies for Radarr. Just get your real movies location cleaned up and neat and then do a bulk import on it. I suggest `./Movie Title (Year)/Movie Title (Year) Edition [Quality-Resoluiton]-GrouP.ext`, though dropping `-GrouP` will make it neater and have little impact. But Quality and Edition are irreplaceable data you should store in the file name.

If what you're *really* after is hard linking a torrent to a movie file for seeding, I'd look into duplicate finders like `fdupes` and `ddupe`, they can generally find duplicate files and point them out or even make them into hard links.
